src/cortex_cli/cortex_cli.py

```diff
@@ -553,6 +545,15 @@ def login(  # pylint: disable=too-many-arguments, too-many-locals, too-many-bran
     """Authenticate on the IQM server, and optionally start a token manager to maintain the session."""
     _set_log_level_by_verbosity(verbose)
 
+    if platform.system().lower().startswith('win') and not no_refresh:
+        click.echo(
```

Comment:
This will also be shown in case the user sets `--no-daemon` flag, is this intended?

Reply:
I was thinking that maybe it's useful to see the message about having to keep the terminal session open. But I guess if the user chooses the `--no-daemon` flag they should already know what it means. And in non-Windows case this message is not shown anyway, so the way I have it here is not consistent. Changed it now.